src/Cli/dotnet/ReleasePropertyProjectLocator.cs

```diff
@@ -129,7 +131,10 @@ public IEnumerable<string> GetCustomDefaultConfigurationValueIfSpecified()
                     }
                     catch (GracefulException)  // Fall back to looking for a solution if multiple project files are found, or there's no project in the directory.
                     {
-                        string? potentialSln = Directory.GetFiles(arg, "*.sln", SearchOption.TopDirectoryOnly).FirstOrDefault();
+                        string? potentialSln = ((string[])[
```

Comment:
You could probably avoid making an array here by searching for FirstOrDefault on the first one then ?? with the second. Also, should we look for slnf, too? You don't have to have your .sln next to your .slnf (though it's common)

Reply:
I am not sure if this does/should support .slnf files